client/my-sites/checkout/checkout-thank-you/akismet-checkout-thank-you.tsx

```diff
@@ -37,6 +42,78 @@ const AkismetCheckoutThankYou: FunctionComponent< AkismetCheckoutThankYouProps >
 		);
 	};
 
+	const userActivePurchases = useSelector(
+		( state ) => getUserPurchases( state )?.filter( ( purchase ) => purchase.active ) ?? []
+	);
+
+	const { overlapMessage, thanksHeadline, thanksMessage } = useMemo( () => {
+		const akismetProducts = AKISMET_PRODUCTS_LIST as ReadonlyArray< string >;
+		const akismetPurchases = userActivePurchases.filter(
+			( purchase ) => isAkismetProduct( purchase ) && 'siteless.akismet.com' === purchase.domain
```

Comment:
Not a blocker, but you could use `isAkismetTemporarySitePurchase( purchase )` instead of `'siteless.akismet.com' === purchase.domain` here..  But it's doing the same thing anyway.  Just pointing out that there is a function already for it. 😉

```suggestion
			( purchase ) => isAkismetProduct( purchase ) && isAkismetTemporarySitePurchase( purchase )
```

Reply:
Good suggestion - thanks! I'll get that updated.